core/citrus-spring/src/main/java/org/citrusframework/reporter/ReporterConfig.java

```diff
@@ -33,10 +40,12 @@ public class ReporterConfig {
     public static final String DEFAULT_JUNIT_REPORTER_ENABLED_PROPERTY = "citrus.default.junit.reporter.enabled";
     public static final String DEFAULT_HTML_REPORTER_ENABLED_PROPERTY = "citrus.default.html.reporter.enabled";
 
+    public static final String DEFAULT_LOGGING_REPORTER_PRINT_STACK_TRACES_PROPERTY = "citrus.default.logging.reporter.print-stack-traces";
```

Comment:
I'd suggest to move this setting to CitrusSettings in order to also support users that are not using Spring